post_install.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+
+destdir_prefix = os.environ['MESON_INSTALL_DESTDIR_PREFIX']
+libdir = os.path.join(destdir_prefix, sys.argv[1])
+suffix = sys.argv[2]
+major_version = sys.argv[3]
+
+try:
+    os.symlink('libopenh264.{}'.format(suffix), os.path.join(libdir, 'libopenh264.so.{}'.format(major_version)))
```

Comment:
Because openh264 wants to make two symlinks anyway, the final output of make install there was:
```
├── include
│   └── wels
│       ├── codec_api.h
│       ├── codec_app_def.h
│       ├── codec_def.h
│       └── codec_ver.h
└── lib
    ├── libopenh264.a
    ├── libopenh264.so -> libopenh264.so.4
    ├── libopenh264.so.1.7.0
    ├── libopenh264.so.4 -> libopenh264.so.1.7.0
    └── pkgconfig
        └── openh264.pc
```

`ninja install` produces the exact same layout with these definitions :)